### PR TITLE
ADCIRC Use Case 2 - Updated Template

### PR DESCRIPTION
**Pull request by cdelcastillo21**
Updated usecase2.md to conform to new agreed upon template. Put additional info and link to Jupyter notebook in 'Implementation section'.

### CODE REVIEW

File: docs/dawson/usecase2.md [hunk 1]
```diff
@@ -178,4 +173,10 @@ Next we want to move ADCIRC inputs/outputs from your Jupyter instance where they
 
 ![Example data relation diagram for an ADCIRC Simulation DataSet](img/curation.png)
 
-Example data relation diagram for an ADCIRC Simulation DataSet
\ No newline at end of file
+Example data relation diagram or an ADCIRC Simulation DataSet
+
+## Citations and Licensing
+
+- Please cite
```
Comment (kks32) [inline docs/dawson/usecase2.md]: I think you are missing citation!

**Review by kks32 — CHANGES_REQUESTED** [review]
Please fix the citation it's missing a description

File: docs/dawson/usecase2.md (hunk 1 above)
Comment (cdelcastillo21) [reply docs/dawson/usecase2.md]: You're totally right!  Just added them!

File: docs/dawson/usecase2.md [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
 
-## Overview
+## Resources
+
+- [DesignSafe ADCIRC Application](https://www.designsafe-ci.org/rw/workspace/#!/ADCIRC::Simulation).
+- [Jupyter nebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis).
+- [ADCIRC Wiki](https://wiki.adcirc.org/Main_Page)
+- [ADCIRC Web Page](https://adcirc.org/)
+- [pyADCIRC GitHub Repository](https://github.com/UT-CHG/pyadcirc) - pyADCIRC documentation.
+- [Creating an ADCIRC DataSet.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/ADCIRC)
```
Comment (kks32) [inline docs/dawson/usecase2.md]: Sorry one more change, could you point to specific notebooks like these:

https://deploy-preview-45--ds-use-case-preview.netlify.app/brandenberg/apiusecases/usecase/

Here's a sample link configuration that can directly open notebooks in Jupyter:

https://jupyter.designsafe-ci.org/hub/user-redirect/notebooks/CommunityData/Use%20Case%20Products/APIs/api_background.ipynb

**Review by kks32 — CHANGES_REQUESTED** [review]

File: docs/dawson/usecase2.md (hunk 2 above)
Comment (cdelcastillo21) [reply docs/dawson/usecase2.md]: I think I got them now working.

**Review by kks32 — APPROVED** [review]
Perfect! Thank you Carlos

**Pull request merged**